game/textinput.hh

```diff
@@ -9,19 +9,19 @@ struct TextInput {
 	/// appends unicode symbol
 	TextInput& operator+=(unsigned int ucs) { //don't know if this is still relevant since SDL2 handles unicode
 		if (ucs < 0x80) {
-			text += ucs;
+			text += static_cast<std::basic_string<char>::value_type>(ucs);
```

Comment:
looks strange, I could understand std::string::value_type or char directly, but as the template param IS the value_type, seem to be complicated here...

Reply:
I just took what the compiler suggested there. I want to revise this code in a later PR because I'm not completely sure most of it is actually necessary.